src/WebJobs.Script/Utility.cs

```diff
@@ -974,6 +977,15 @@ public static FunctionAppContentEditingState GetFunctionAppContentEditingState(I
             }
         }
 
+        /// <summary>
+        /// Gets a timestamp given start ticks (from <see cref="Stopwatch.GetTimestamp"/>) without allocating a Stopwatch.
+        /// </summary>
+        public static TimeSpan GetDuration(long startTicks)
```

Comment:
Why not have it return milliseconds directly instead of creating a `TimeSpan` just to read the `TotalMilliseconds` ?

Reply:
Good question! This was for consistency if we went with one thing across projects, because of the direct use of `TimeSpan` here: https://github.com/Azure/azure-webjobs-sdk/pull/2794/files#diff-673ae62ecb874d5346bc217f774d055a5da4728f418a2e169028d205428dd020R149 - I'd appreciate any thoughts!

Reply:
I think the main value is to encapsulate the call to `GetTimeStamp()` and the conversion between ticks and ms. So maybe create a `TimeSpan` from the caller when you need one, with `new TimeSpan(GetDuration(ticks))`.

But it's not much, didn't realize it was used somewhere else.

Reply:
Getting back to this - pulling in `ValueStopwatch` overall and globally replacing!